### PR TITLE
facets: experiments facets fix

### LINKED ISSUE #792: receivers: match_valid_experiments problem
```shell
In [3]: from inspirehep.modules.records.receivers import match_valid_experiments

In [4]: match_valid_experiments('1', {u'accelerator_experiments': [{u'curated_relation': False, u'experiment': [u'MAGIC', u'MAGIC']}]})
```

```traceback
Traceback (most recent call last):
  File "/opt/inspire/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/opt/inspire/lib/python2.7/site-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/inspire/lib/python2.7/site-packages/invenio_records/tasks/index.py", line 47, in index_record
    before_record_index.send(recid, json=json, index=index)
  File "/opt/inspire/lib/python2.7/site-packages/blinker/base.py", line 267, in send
    for receiver in self.receivers_for(sender)]
  File "/opt/inspire/src/inspire/inspirehep/modules/records/receivers.py", line 95, in match_valid_experiments
    experiment = experiment.lower()
AttributeError: 'list' object has no attribute 'lower'
```

The receiver should be updated to handle the case where `experiment` key is a list.

### PR DESCRIPTION
**Pull request by Panos512**
* Experiments matching now also support multiple experiments for a record.

* Experiments facets are now correctly displayed.

* Closes #792 .

Signed-off-by: Panos Paparrigopoulos <panos.paparrigopoulos@cern.ch>

### CODE REVIEW

File: inspirehep/modules/records/testsuite/test_records.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of INSPIRE.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2015, 2016 CERN.
```
Comment (jmartinm) [inline inspirehep/modules/records/testsuite/test_records.py]: Shall we move this test to the top level `tests/` folder? You can create a subfolder `records` there if we want to better separate tests by functionality.

**Comment by jmartinm** [comment]
Apart from the small comment, this PR is ready? I can test it locally and if ok merge and deploy on QA.

**Comment by Panos512** [comment]
Yeah it should work

**Comment by jmartinm** [comment]
Sorry for the delay @Panos512 , can you rebase this branch to merge it?

**Pull request merged**